os/customizing-docker.md

```diff
@@ -95,13 +93,43 @@ cp -p ~/cfssl/client.pem cert.pem
 cp -p ~/cfssl/client-key.pem key.pem
 ```
 
-### Drop-in Configuration
+### Enable the Secure Remote API on a New Socket
+
+**NOTE:** Skip this part if you use CoreOS releases older than 949.0.0.
```

Comment:
These docs are versioned so we should be able to say "for older releases, you must X, follow this guide"